### ISSUE TITLE
Element with aria-hidden has no content in sequential focus navigation - Description of Failed Example 6 is not clear

### ISSUE THREAD

**Issue by giacomo-petri** [auto-wcag/auto-wcag#2409]
The description of [Failed Example 6](https://www.w3.org/WAI/standards-guidelines/act/rules/6cfa84/#failed-example-6) sounds a bit weired:

> This a element is not [focusable](https://www.w3.org/WAI/standards-guidelines/act/rules/6cfa84/#focusable) because it moves focus to the input element whenever it receives focus.

If read together with [Passed Example 4](https://www.w3.org/WAI/standards-guidelines/act/rules/6cfa84/#passed-example-4), it makes sense, but on its own it's not very clear. In particular, the first sentence seems to suggest that the element is focusable because it doesn't move focus away, which is ok based on our definition of "focusable", but can be misleading.
